.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Describe your changes
+< Summary of the changes.>
+
+< Please also include relevant motivation and context. >
+
+< List any dependencies that are required for this change. >
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist before requesting a review
+
+- [ ] My branch is up-to-date with the target branch - if not update your fork with the changes from the target branch (use `pull` with `--rebase` option if possible).
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have requested a reviewer and an assignee (assignee is responsible for merging)
+
```

Comment:
```suggestion
- [ ] I have given the PR a name that clearly describes the changes, written in imperative form ([context](https://www.gitkraken.com/learn/git/best-practices/git-commit-message#using-imperative-verb-form).
```

My idea here is that we should be able to use the PR title directly as the commit message when merging (so the assignee does not have to spend extra time formulating a descriptive commit message when merging). I think it's nice to follow the convention of imperative mood for commit messages.